fog/ingest/enclave/impl/tests/tx_processing.rs

```diff
@@ -82,7 +82,7 @@ fn test_ingest_enclave(logger: Logger) {
         assert_eq!(tx_rows.len(), 10);
 
         // Check that the tx row ciphertexts have the right size
-        const EXPECTED_PAYLOAD_SIZE: usize = 188; // The observed tx_row.payload size
+        const EXPECTED_PAYLOAD_SIZE: usize = 159; // The observed tx_row.payload size
```

Comment:
this shows the change in size of TxOutRecord when it is serialized, so we saved 29 bytes